CorsixTH/Lua/app.lua

```diff
@@ -1517,6 +1517,20 @@ function App:quickLoad()
   end
 end
 
+--! Function to check the savegame version matches or is older than the installation of CorsixTH
+--!param save_version (num)
+--!return true if compatible, otherwise false
+function App:checkCompatibility(save_version)
```

Comment:
Abstraction level of 1520 doesn't match with function name.

It should describe what "compatible" means (ie loaded game can be run in the program), rather than how you compute it. If you find a newer or better way to decide compatibility, the function description shouldn't need to change.